AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/allocate-hearings/allocate-hearings.component.ts

```diff
@@ -44,6 +52,7 @@ export class AllocateHearingsComponent implements OnInit {
     customIconTransform: Transform = { rotate: 45 };
     private filterSize = 20;
     dropDownUserLabelAllocateTo = 'Allocate to';
+    todayDate: string;
```

Comment:
Can this be a Date?

Reply:
NO. It needs to be a string as it is used to populate the input date box